rest/specs/src/main/filtered-resources/openapi/openapi.yaml

```diff
@@ -169,10 +169,30 @@ components:
       type: string
       minLength: 1
       maxLength: 256
-    Metacard:
+    Metadatas:
       description: >
-        A metacard.xml attachment that is sent along with a product to enhance the product's discoverability. The
-        current maximum is the character equivalent of 10GB.
+        A list of metadata objects that a client can pass along with a product to enrich indexing and query
+        capabilities.
+      type: array
+      items:
+        $ref: '#/components/schemas/Metadata'
+    Metadata:
+      description: An object that contains the metadata type and the metadata file.
+      type: object
+      properties:
+        metadataType:
+          $ref: '#/components/schemas/MetadataType'
+        metadataContent:
+          $ref: '#/components/schemas/MetadataContent'
+    MetadataType:
+      description: >
+        The type of metadata that is being sent.
+      type: string
+      minLength: 1
+      maxLength: 256
+    MetadataContent:
+      description: >
+        The metadata file attachment. The current maximum is the character equivalent of 10GB.
       type: string
       format: binary
       minLength: 1
```

Comment:
❓ Should there be a max?

Reply:
Shouldn't we limit metadata file size like we do with the product file?

Reply:
My comment was to the fact that I couldn't see a `maxLength` defined in the spec

Reply:
never mind ... just noticed I didn't expand the rest ... doh